tailnet/telemetry.go

```diff
@@ -31,6 +33,19 @@ type TelemetryStore struct {
 
 	cleanDerpMap  *tailcfg.DERPMap
 	cleanNetCheck *proto.Netcheck
+	nodeIDSelf    uint64
+	homeDerp      int32
+	application   string
+
+	// nil if not connected
+	connSetupTime *durationpb.Duration
+	connectedIP   *netip.Addr
```

Comment:
To support coderd and agent telemetry in the future, we'll need to change this to a map of peers to nodes/IPs.